Formula/openal-soft.rb

```diff
@@ -19,11 +19,15 @@ class OpenalSoft < Formula
   depends_on "portaudio" => :optional
   depends_on "pulseaudio" => :optional
   depends_on "fluid-synth" => :optional
+  depends_on "jack" => :optional
 
   # clang 4.2's support for alignas is incomplete
   fails_with(:clang) { build 425 }
 
   def install
+    #  See: https://github.com/kcat/openal-soft/issues/153
+    ENV.append "LDFLAGS", "-Wl,-rpath,#{opt_lib}"
```

Comment:
What does this fix exactly?

Reply:
`openal-info` runs now.

```
$ brew info openal-soft
openal-soft: stable 1.18.2 (bottled), HEAD [keg-only]
Implementation of the OpenAL 3D audio API
http://kcat.strangesoft.net/openal.html
/usr/local/Cellar/openal-soft/1.18.2 (30 files, 949.3KB)
  Built from source on 2017-10-08 at 23:27:28 with: --with-fluid-synth --with-jack --with-portaudio --with-pulseaudio
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/openal-soft.rb
==> Dependencies
Build: pkg-config ✔, cmake ✔
Optional: portaudio ✔, pulseaudio ✔, fluid-synth ✔, jack ✔
==> Options
--with-fluid-synth
	Build with fluid-synth support
--with-jack
	Build with jack support
--with-portaudio
	Build with portaudio support
--with-pulseaudio
	Build with pulseaudio support
--HEAD
	Install HEAD version
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local,
because macOS provides OpenAL.framework.

If you need to have this software first in your PATH run:
  echo 'export PATH="/usr/local/opt/openal-soft/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/openal-soft/lib
    CPPFLAGS: -I/usr/local/opt/openal-soft/include
For pkg-config to find this software you may need to set:
    PKG_CONFIG_PATH: /usr/local/opt/openal-soft/lib/pkgconfig
```
```
$ /usr/local/Cellar/openal-soft/1.18.2/bin/openal-info
Available playback devices:
W: [] caps.c: Normally all extra capabilities would be dropped now, but that's impossible because PulseAudio was built without capabilities support.
    CoreAudio Default
Available capture devices:
    CoreAudio Default
Default playback device: CoreAudio Default
Default capture device: CoreAudio Default
ALC version: 1.1

** Info for device "CoreAudio Default" **
ALC version: 1.1
ALC extensions:
    ALC_ENUMERATE_ALL_EXT, ALC_ENUMERATION_EXT, ALC_EXT_CAPTURE,
    ALC_EXT_DEDICATED, ALC_EXT_disconnect, ALC_EXT_EFX,
    ALC_EXT_thread_local_context, ALC_SOFTX_device_clock, ALC_SOFT_HRTF,
    ALC_SOFT_loopback, ALC_SOFT_output_limiter, ALC_SOFT_pause_device
No HRTFs found
OpenAL vendor string: OpenAL Community
OpenAL renderer string: OpenAL Soft
OpenAL version string: 1.1 ALSOFT 1.18.2
OpenAL extensions:
    AL_EXT_ALAW, AL_EXT_BFORMAT, AL_EXT_DOUBLE, AL_EXT_EXPONENT_DISTANCE,
    AL_EXT_FLOAT32, AL_EXT_IMA4, AL_EXT_LINEAR_DISTANCE, AL_EXT_MCFORMATS,
    AL_EXT_MULAW, AL_EXT_MULAW_BFORMAT, AL_EXT_MULAW_MCFORMATS, AL_EXT_OFFSET,
    AL_EXT_source_distance_model, AL_EXT_SOURCE_RADIUS, AL_EXT_STEREO_ANGLES,
    AL_LOKI_quadriphonic, AL_SOFT_block_alignment, AL_SOFT_deferred_updates,
    AL_SOFT_direct_channels, AL_SOFT_gain_clamp_ex, AL_SOFT_loop_points,
    AL_SOFT_MSADPCM, AL_SOFT_source_latency, AL_SOFT_source_length,
    AL_SOFT_source_resampler, AL_SOFT_source_spatialize
Available resamplers:
    Nearest
    Linear *
    4-Point Sinc
    Band-limited Sinc (12/24)
EFX version: 1.0
Max auxiliary sends: 2
Supported filters:
    Low-pass, High-pass, Band-pass
Supported effects:
    EAX Reverb, Reverb, Chorus, Distortion, Echo, Flanger, Ring Modulator,
    Compressor, Equalizer, Dedicated Dialog, Dedicated LFE
```